src/app/services/GameService.ts

```diff
@@ -35,21 +35,21 @@ export class GameService implements OnDestroy {
 
     private followedPartId: MGPOptional<string> = MGPOptional.empty();
 
-    private followedPartObs: MGPOptional<Observable<ICurrentPartId>> = MGPOptional.empty();
+    private followedPartObs: MGPOptional<Observable<MGPOptional<IPart>>> = MGPOptional.empty();
```

Comment:
Mmh, le premier Optional parce que on est peut être pas encore abonné.
Le deuxième c'est pourquoi ?

Reply:
que le document existe ! donc 
1. empty() signifie qu'on est pas abonné
2. of(observable(emtpy()) signifie qu'on est abonné et qu'il existe pas ?

Reply:
exactement !

Reply:
ça vaudrait la peine d'être indiqué dans un /** */ précédant la variable !